cli/azd/pkg/tools/azcli/container_registry.go

```diff
@@ -150,3 +206,56 @@ func (crs *containerRegistryService) createRegistriesClient(
 
 	return client, nil
 }
+
+func (crs *containerRegistryService) getAcrToken(
+	ctx context.Context,
+	subscriptionId string,
+	loginServer string,
+) (*acrToken, error) {
+	creds, err := crs.credentialProvider.CredentialForSubscription(ctx, subscriptionId)
+	if err != nil {
+		return nil, fmt.Errorf("getting credentials for subscription '%s': %w", subscriptionId, err)
+	}
+
+	token, err := creds.GetToken(ctx, policy.TokenRequestOptions{Scopes: auth.LoginScopes})
+	if err != nil {
+		return nil, fmt.Errorf("getting token for subscription '%s': %w", subscriptionId, err)
+	}
+
+	formData := url.Values{}
+	formData.Set("grant_type", "access_token")
+	formData.Set("service", loginServer)
+	formData.Set("access_token", token.Token)
```

Comment:
Based on the docs, should we set `tenant` explicitly? The docs seem to call out the specific requirement, although the access token here should have the correct `tid` claim set.

Reply:
i did see that tenant was included in the docs but it works without it and didn't want to have to bring in the multi tenant auth provider, etc if we didn't need to.